docs/real-time-analytics/filter-logs-for-splunk/index.md

```diff
@@ -0,0 +1,154 @@
+---
+seo:
+  title: Filtering Confluent logs for Splunk
+  description: This recipe demonstrates how to filter Confluent audit logs to Splunk for SIEM processing
+---
+
+# Filter Kafka Audit Logs for output to Splunk
+
+In the Security Information and Event Management (SIEM) world, it's just as important to have insight into internal activities as it is to monitor for external security threats and vulnerabilities. But viewing all internal audit logs would provide too much information; you need to narrow the scope to particular events.
+
+This recipe demonstrates how to filter audit logs in a Kafka topic and filter them for specific events and forward them to Splunk for indexing via the [Splunk Sink connector](https://docs.confluent.io/cloud/current/connectors/cc-splunk-sink.html#cc-splunk-sink).  The example data used is from [Confluent Cloud](https://www.confluent.io/confluent-cloud/tryfree/?utm_source=github&utm_medium=ksqldb_recipes&utm_campaign=filter_logs_for_splunk) audit logs.
+The stream processing application filters for events involving operations on `topics`, but you can review the [structure of Confluent audit logs](https://docs.confluent.io/platform/current/security/audit-logs/audit-logs-concepts.html#audit-log-content) and extend this solution to filter for any [auditable event](https://docs.confluent.io/platform/current/security/audit-logs/audit-logs-concepts.html#auditable-events).
+
+!!! note "Recipe Considerations" 
```

Comment:
@ybyzek added this section describing assumptions

Reply:
Perfect, thank you!